apiserver/common/addresses.go

```diff
@@ -15,7 +17,7 @@ import (
 // controller addresses and the CA public certificate.
 type AddressAndCertGetter interface {
```

Comment:
This interface name is now a bit inaccurate

Reply:
I've renamed it, although the types in this file aren't very well scoped anyway. CA certs have bugger all to do with addresses, and what's the ModelUUID doing in there? Anyway.